lib/matplotlib/animation.py

```diff
@@ -957,7 +957,7 @@ def _stop(self, *args):
 
     def save(self, filename, writer=None, fps=None, dpi=None, codec=None,
              bitrate=None, extra_args=None, metadata=None, extra_anim=None,
-             savefig_kwargs=None):
+             savefig_kwargs=None, progress_callback=None):
```

Comment:
Perhaps make the argument keyword-only?

Reply:
Just to be clear, this needs `savefig_kwargs=None, *, progress_callback=None)` to make kwarg only....

Reply:
Done